Push multiple branches in one call to assign

LinuxMercedes opened the pull request:
Closes #77 (and fixes part of #59).

brhoades commented on assigner/baserepo.py:
```diff
@@ -176,6 +176,10 @@ def clone_to(self, dir_name, branch=None):
                 self._repo = git.Repo.clone_from(self.ssh_url, dir_name,
                                                 branch=branch)
+                for b in branch:
+                    self._repo.create_head(b, 'origin/{}'.format(b))
```
np: we should stick to double quotes, I think that's what the rest of the file is using.

brhoades commented:
This looks good, let's just make it consistent.

brhoades commented on assigner/commands/assign.py:
```diff
@@ -64,7 +64,8 @@ def assign(conf, args):
                     repo = StudentRepo.new(base, semester, student_section,
                                            username, token)
                     repo.push(base, branch)
```
and repo.push is ok with a list?

LinuxMercedes replied:
Yep, it will push multiple branches if you ask it to!

LinuxMercedes commented:
Since commas are valid in git branch names, I switched from the hacky way of splitting up the command line list  to using argparse the way it wants to be used. (Spaces aren't valid in git branch names, so space-delimiting works great.)

Also updated the tutorial to explain how to use this feature.
